src/pages/index.js

```diff
@@ -29,30 +35,38 @@ const initialCards = [
 ];
 
 // Profile Elements
-const profileEditButton = document.querySelector(".profile__edit-button");
-const editProfileModal = document.querySelector("#edit-profile-modal");
-const profileCloseButton = document.querySelector("#close-profile-modal");
-const profileFormElement = document.querySelector("#edit-profile-form");
-const profileNameInput = document.querySelector("#edit-profile-input-name");
-const profileJobInput = document.querySelector(
-  "#edit-profile-input-description"
-);
-const profileNameElement = document.querySelector(".profile__title");
-const profileJobElement = document.querySelector(".profile__description");
+const profileElements = {
+  editButton: document.querySelector(".profile__edit-button"),
+  modal: document.querySelector("#edit-profile-modal"),
+  closeButton: document.querySelector("#close-profile-modal"),
+  form: document.querySelector("#edit-profile-form"),
+  nameInput: document.querySelector("#edit-profile-input-name"),
+  jobInput: document.querySelector("#edit-profile-input-description"),
+  nameElement: document.querySelector(".profile__title"),
+  jobElement: document.querySelector(".profile__description"),
```

Comment:
**Could be improved**

We can get any form using their **unique** attribute `name` or `id` from `document.forms`

```js

const profileForm = document.forms["profile-form"];
const cardForm = document.forms["card-form"];

```

You need to add `name` or `id` first in `index.html` to achieve that.

More info here https://developer.mozilla.org/en-US/docs/Web/API/Document/forms